internal/diagnostics/descriptions/parsers/cssParser.ts

```diff
@@ -14,7 +14,14 @@ export const cssParser = createDiagnosticsCategory({
 	EXPECTED_IDENTIFIER: {message: markup`Expected an identifier`},
 	EXPECTED_SELECTOR: {message: markup`Expected a selector`},
 	EXPECTED_LBRACKET: {
-		message: markup`Expected a left bracket </emphasis>{</emphasis>`,
+		message: markup`Expected a left bracket <emphasis>{</emphasis>`,
 	},
 	UNEXPECTED_EMPTY_SELECTOR: {message: markup`Unexpected empty selectors`},
+	EXPECTED_CLOSING_ATTRIBUTE_SELECTOR: {
+		message: markup`Expected to close attribute selector with a right bracket <emphasis>]</emphasis>`,
```

Comment:
```suggestion
		message: markup`Expected to close attribute selector with a right square bracket <emphasis>]</emphasis>.`,
```

We could add the character we got instead of the square bracket